sentry-core/src/main/java/io/sentry/core/Sentry.java

```diff
@@ -294,7 +346,7 @@ public static void setTransaction(@Nullable String transaction) {
    *
    * @param user the user
    */
-  public static void setUser(@Nullable User user) {
+  public static void setUser(final @Nullable User user) {
```

Comment:
Super hard to review this PR with so much unrelated changes like this

Reply:
true, I should not have changed this, but it's a Draft right